derive-tag/src/lib.rs

```diff
@@ -0,0 +1,69 @@
+use proc_macro::TokenStream;
+use quote::quote;
+use syn;
+
+#[proc_macro_derive(ToTag)]
+pub fn to_tag_derive(input: TokenStream) -> TokenStream {
+    let ast = syn::parse(input).unwrap();
```

Comment:
Instead of a plain `.unwrap` I would recommend using a `.unwrap_or_else(|e| e.to_compile_error())`, so the error is shown better (I think...).

I've tried it locally and it could look like this:
```rust
    syn::parse(input)
        .map(|ast| impl_to_tag(&ast))
        .unwrap_or_else(|e| e.to_compile_error().into())
```

I've seen this pattern in some very popular derive crates out there, and I've done in mine as well 😅